src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -93,6 +93,9 @@ export function transformVAOSAppointment(appt) {
     appointmentType === APPOINTMENT_TYPES.request ||
     appointmentType === APPOINTMENT_TYPES.ccRequest;
   const providers = appt.practitioners;
+  const cTreatingSpecialty = !appt.extension?.ccTreatingSpecialty
```

Comment:
Can we name this variable `ccTreatingSpecialty`. Also, what do you think about simplifying this to: 
`const ccTreatingSpecialty = appt.extension?.ccTreatingSpecialty;`

Reply:
Thanks, good catch.